### DISCUSSION TITLE
Publish to Nix package manager

### DISCUSSION THREAD

**Discussion by matttrach** [getgauge/gauge#2196] [Ideas]
It would really great if you published releases to Nix for use with the Nix package manager or NixOS.
https://pkgs.on-nix.com/nixpkgs/gauge/

The current latest version is 1.1.6, I would like to use the latest releases by adding them to my Nix derivations.

**Comment by sriv** [comment]
The nix package is not maintained by us, it is best if you can reach out to the package maintainer.

Looking at the package config [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/gauge/default.nix) it seems like a PR could be triggered as part of the gauge release pipeline, but we wouldn't want to do that without the maintainer's consent.

**Comment by matttrach** [comment]
vdemeest@redhat.com @vdemeester is the maintainer it seems, I wonder how hard it would be to get them to see this.

**Comment by vdemeester** [comment]
Ah my bad.. Usually, I have a rss feed of releases to know when a new one is done so that I can update the package. For some reason, I did not see the gauge ones 😓 

> Looking at the package config here it seems like a PR could be triggered as part of the gauge release pipeline, but we wouldn't want to do that without the maintainer's consent.

That could work, if you create a PR updating a package, it will notify the maintainer to get approval, so automating this could definitely work (it wouldn't get auto-merged, but still, it would make it even easier). 
Note there is also a bot on nixpkgs that can auto-upgrade packages, I wonder why it never picked up gauge yet.

**Comment by sriv** [comment]
Thanks for checking this @vdemeester . If there's anything we can do from gauge's end to help please do let me know. I guess we'd prefer to stay with whatever the nix team already has in place (ex - the auto upgrade bot)?
